x-pack/plugins/ingest_manager/server/services/epm/registry/index.ts

```diff
@@ -39,6 +41,17 @@ export async function fetchList(params?: SearchParams): Promise<RegistrySearchRe
   return fetchUrl(url.toString()).then(JSON.parse);
 }
 
+export async function fetchLatestPackage(
+  packageName: string,
+  internal: boolean = false
+): Promise<RegistrySearchResult> {
+  const registryUrl = appContextService.getConfig()?.epm.registryUrl;
+  const url = new URL(`${registryUrl}/search?package=${packageName}&internal=${internal}`);
+  const searchResults = await fetchUrl(url.toString());
+  const latestPackage = JSON.parse(searchResults);
+  return latestPackage[0];
```

Comment:
maybe we can do a check here that there is an item returned by the search? and throw a Error if there is not?